diffpy/labpdfproc/tests/test_set_distances_at_angle.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from diffpy.labpdfproc.functions import Gridded_circle
+
+
+def test_set_distances_at_angle():
+    actual_gs = Gridded_circle(radius=1, n_points_on_diameter=3, mu=1)
+
+    # tests for acute angles
```

Comment:
when you do multiple tests it is better to use the `@pytest.mark.paramatrize()` decorator on the test function and run different examples as a list of tuples.  You may be able to google how to do this.